### ISSUE TITLE
[Python] Builtin function not supported in transform

### ISSUE THREAD

**Issue by ddebrunner** [IBMStreams/streamsx.topology#302]
I tried to simplify the sample transform to use int() directly rather than string_to_int

I got this error:
```
[streamsadmin@streamsqse modules]$ python3 myapp.py
Traceback (most recent call last):
  File "myapp.py", line 7, in <module>
    i1 = source.transform(int)
  File "/home/streamsadmin/topology_pr/streamsx.topology/com.ibm.streamsx.topology/opt/python/packages/streamsx/topology/topology.py", line 102, in transform
    op = self.topology.graph.addOperator("com.ibm.streamsx.topology.functional.python::PyFunctionTransform", func)
  File "/home/streamsadmin/topology_pr/streamsx.topology/com.ibm.streamsx.topology/opt/python/packages/streamsx/topology/graph.py", line 31, in addOperator
    self.resolver.add_dependencies(inspect.getmodule(function))
  File "/home/streamsadmin/topology_pr/streamsx.topology/com.ibm.streamsx.topology/opt/python/packages/streamsx/topology/dependency.py", line 20, in add_dependencies
    self._add_dependency(module)
  File "/home/streamsadmin/topology_pr/streamsx.topology/com.ibm.streamsx.topology/opt/python/packages/streamsx/topology/dependency.py", line 54, in _add_dependency
    module_path = os.path.abspath(module.__file__)
AttributeError: module 'builtins' has no attribute '__file__'
```

**Comment by henrychi2** [comment]
Must-fix for alpha

**Comment by henrychi2** [comment]
This is a bug; built-in functions should be supported.  Fix is included in #313.
